test/test_header_emitter.js

```diff
@@ -305,6 +305,105 @@ suite('headeremitter', function () {
       assert.equal(now.getTime() - now.getMilliseconds(), reparsed.getTime());
     });
   });
+
+  /**
+     * Test addDate method when displayUTC flag is set. This is to support add-ons which dont want 
+     * emails to contain TimeStamp information when email is sent. For example, Torbirdy add-on doesnt 
+     * want timestamp to be shown as it can leak the user location.
+     */
```

Comment:
You don't need a documentation comment for this test suite--if the test suite name isn't indicative of what's being tested, then something's wrong.